alpha/lib/model/DeliveryProfileLive.php

```diff
@@ -442,13 +445,41 @@ protected function filterStreamIdsByBitrate ($streams)
 		
 		if (!count($streamIds))
 		{
-			// If the stream info is available to us, min and/or max bitrate params were passed, and no streams comply with them - throw error
-			KExternalErrors::dieError(KExternalErrors::FLAVOR_NOT_FOUND, "Entry [". $this->getDynamicAttributes()->getEntryId() ."] has no valid streams");
+			// If the stream info is available to us, min and/or max bitrate params were passed, and no streams comply with them - issue warning and cancel restrictions
+			KalturaLog::warning("Entry [". $this->getDynamicAttributes()->getEntryId() ."] has no streams which comply with the min/max bitrate limitations.");
```

Comment:
Use single quote instead of double since not processing is needed.

Reply:
resolved.